server/server.go

```diff
@@ -38,16 +40,20 @@ type HTTPServer struct {
 	Content content.RuleContentDirectory
```

Comment:
wrong indentation?

Reply:
hmm nope, just GitHub is having issue to show it correctly